os/include/tinyara/log_dump/log_dump.h

```diff
@@ -19,21 +19,22 @@
 /****************************************************************************
  * Included Files
  ****************************************************************************/
+
 #include <stdio.h>
 #include <fcntl.h>
+#include <unistd.h>
 
 /********************************************************************************
  * Definitions
  ********************************************************************************/
 
-/* Log Dump Thread information */
-#define LOG_DUMP_NAME        "log_dump"		/* Log dump thread name */
-#define LOG_DUMP_STACKSIZE   16384		/* Log dump thread stack size */
-#define LOG_DUMP_PRIORITY    100		/* Log dump thread priority */
-
 #define LOGDUMP_SAVE_START	"1"
 #define LOGDUMP_SAVE_STOP	"2"
 
+#define LOG_DUMP_OK		0	/* Used for successful return check */
+#define LOG_DUMP_OPT_FAIL	-2
```

Comment:
There are definitions for failures like LOG_DUMP_XXX_FAIL in multiple files, os/include/tinyara/log_dump/log_dump.h and os/kernel/log_dump/log_dump.h
How about putting them in one place to avoid confusion?

Reply:
The header file in os/include/tinyara/log_dump/log_dump.h can be used any thread while the header file in os/kernel/log_dump/log_dump.h is only used internally.
So, I seperated the definitions.